### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by algonell**
Fix typos in documentation, comments, etc.

### CODE REVIEW

File: _posts/2016-03-16-edition-13.markdown [hunk 1]
```diff
@@ -298,7 +298,7 @@ __Light reading__
 * [Git rebase and the golden rule explained](https://medium.freecodecamp.com/git-rebase-and-the-golden-rule-explained-70715eccc372#.93ok9bhsw), by Pierre de Wulf
 * [GitHub Workflow (used by Frameworks team at BBC News)](http://www.integralist.co.uk/posts/github-workflow.html)
 * [Write, Review, Merge, Publish: Phabricator Review Workflow](https://secure.phabricator.com/phame/post/view/766/write_review_merge_publish_phabricator_review_workflow/)
-* [A succesful Git branching model considered harmful](https://barro.github.io/2016/02/a-succesful-git-branching-model-considered-harmful/), by Jussi Judin
+* [A successful Git branching model considered harmful](https://barro.github.io/2016/02/a-successful-git-branching-model-considered-harmful/), by Jussi Judin
```
Comment (chriscool) [inline _posts/2016-03-16-edition-13.markdown:301]: The change in the URL breaks the link here. Even if it's a typo, we shouldn't make the change in the URL.

Comment (chriscool) [reply _posts/2016-03-16-edition-13.markdown:301]: Also the title of the linked article has the typo too, so not sure we actually should make the change in the title too.

**Comment by chriscool** [comment]
Thanks @algonell for the great fixes! I found a single issue with them, and also it looks like your branch should be rebased.

I can help with fixing the issue and rebasing if you prefer. Just let me know.

Thanks again!

File: _posts/2016-03-16-edition-13.markdown (hunk 1 above)
Comment (algonell) [reply _posts/2016-03-16-edition-13.markdown]: Absolutely, let's keep it as is, thanks!

File: General-Microproject-Information.md [hunk 2]
```diff
@@ -50,8 +50,8 @@ and the implementation of that design were all reviewed and discussed,
 and after several iterations an improved version of the patch was
 accepted into our codebase. As a GSoC contributor, or Outreachy intern,
 you will be playing the role of the developer and engaging in a
-similar discussion. Get familar with the flow, need for clarity on
-both sides (i.e. you need to clearly explain your design, and need to
+similar discussion. Get familiar with the flow, need for clarity on
+both sides (i.e. you need to clearly defend your design, and need to
```
Comment (chriscool) [inline General-Microproject-Information.md:54]: The conflict with the master branch was likely that "defend" has been changed to "explain"  in 89a6e71888e96834bb15aa634f1e431c9b0f36ed, but when you rebased you fixed the conflict by putting back "defend" while I think you should have kept "explain".

Let me know if you want me to fix this.

Comment (chriscool) [reply General-Microproject-Information.md:54]: Except for this issue with the rebase the rest looks like great fixes. Thanks!

Comment (algonell) [reply General-Microproject-Information.md]: My bad, thanks!

File: _posts/2015-05-13-edition-3.markdown [hunk 3]
```diff
@@ -141,7 +141,7 @@ Sébastien's work is very welcome.
 
 Unfortunately git developers can have different views on how to group
 commands together. So it can be difficult for them to agree on such
-kind of changes. Long discussions because of small personnal
+kind of changes. Long discussions because of small personnel
```
Comment (chriscool) [inline _posts/2015-05-13-edition-3.markdown:144]: I think this should actually be "personal". Sorry for not spotting this before.

Comment (chriscool) [reply _posts/2015-05-13-edition-3.markdown:144]: Otherwise everything looks good, thanks! Let me know if you want me to fix this.

Comment (algonell) [reply _posts/2015-05-13-edition-3.markdown]: Absolutely, thanks!

BTW, thanks to your comment I've added an extra one: bikesheedding -> bikeshedding.

**Comment by chriscool** [comment]
LGTM now, so merged. Thanks @algonell !

**Pull request merged**